tests/integ/test_udf.py

```diff
@@ -167,18 +167,20 @@ def int2str(x):
 @pytest.mark.skipif(
     IS_IN_STORED_PROC, reason="Named temporary udf is not supported in stored proc"
 )
-def test_call_named_udf(session, temp_schema, db_parameters):
+def test_call_named_udf(session, temp_schema, db_parameters, local_testing_mode):
     mult_udf_name = Utils.random_name_for_temp_object(TempObjectType.FUNCTION)
-    session._run_query("drop function if exists test_mul(int, int)")
+    if not local_testing_mode:
+        session._run_query("drop function if exists test_mul(int, int)")
     udf(
         lambda x, y: x * y,
         return_type=IntegerType(),
         input_types=[IntegerType(), IntegerType()],
         name=mult_udf_name,
     )
-    Utils.check_answer(
-        session.sql(f"select {mult_udf_name}(13, 19)").collect(), [Row(13 * 19)]
-    )
+    if not local_testing_mode:
+        Utils.check_answer(
+            session.sql(f"select {mult_udf_name}(13, 19)").collect(), [Row(13 * 19)]
+        )
```

Comment:
can this be written into `df.select(call_udf(...)`?

Reply:
There is a `call_udf` below, I think this test case is for SQL specifically.